### ISSUE TITLE
Screen readers are narrating incorrect Role for Multiple controls present on the Iris Copilot dialog.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5396]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader dependent users will face difficulty as screen readers are narrating incorrect Role for multiple controls present on the Iris Copilot dialog so the end-users will get confused, and this may lead to disorientation.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://mcapshelpuat.microsoft.com/

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers.
Step 2: 'MCAPSHelp' Home page will appear. 
Step 3: Navigate to the 'My help' -> 'Account Management Requests' under start button invoked it, ->'Iris Copilot' appeared. 
Step 4: Navigate to the combo box required list items navigate to the buttons using tab key.
Step 5: Verify whether Screen readers are narrating correct Role for Multiple controls or not.

### What do you expect?

Screen readers (NVDA/ JAWS) should narrate correct Role for Multiple controls present on the Iris Copilot dialog while navigating using tab key.
Ex: 1. When focus lands on 'Continue' button, Screen readers should narrate as 'button Continue'.
2..When focus lands on 'Upload Files' button, Screen readers should narrate as 'button Upload Files'.
3.When focus lands on 'Cancel' button, Screen readers should narrate as 'button Cancel'.

### What actually happened?

Screen readers are narrating incorrect Role for Multiple controls present on the Iris Copilot dialog while navigating using tab key.
Observation: 1.When focus lands on 'Continue' button, Screen readers are narrating as 'Continue toggle button not pressed'
2.When focus lands on 'Upload Files' button, Screen readers are narrating as toggle button not pressed Upload Files.
3.When focus lands on 'Cancel' button, Screen readers are narrating as toggle button  not pressed Cancel.

### Do you have any screenshots or recordings to repro the issue?

<img width="1914" alt="Image" src="https://github.com/user-attachments/assets/ef744209-ab03-4ef1-ac99-80a51f6eb837" />
https://github.com/user-attachments/assets/3b347edc-979e-4654-a3a0-16948fabed1c

### MAS reference

https://aka.ms/MAS4.1.2

### Adaptive Card JSON

```json

```

### Additional context

@compulim The reported issue does not repro in Adaptive cards versions 1.3 & 1.6. 
Below is the behavior from 1.3 where screen reader is announcing as Upload files button and Cancel button.
<img width="631" alt="Image" src="https://github.com/user-attachments/assets/eaec3b13-f08b-4e64-9c05-f8113d48a5ed" />

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#BotFrameworkWebChat-Oct24;#WWL:Bug1767726;#WWL:Bug11645870;

**Comment by Kirank915** [comment]
This bug is copied from WWL team: [ISSUE 1767726](https://worldwidelearning.visualstudio.com/WWL%20Accessibility/_workitems/edit/1767726)

**Comment by OEvgeny** [comment]
We use push/toggle buttons due to the lack of better alternatives to resolve a different accessibility/UX issue.

Without the fix, once button is pressed, the chosen option is not visible anymore. To avoid the confusion we convert the buttons to toggle buttons, so user is able to revisit their choice.

I'd say this is a designed behavior. We may revisit this once we get a better design guidance on this topic.

**Comment by Kirank915** [comment]
We have shared feedback with WWL team and closed the bug.
